src/static/riot/competitions/editor/_competition_details.tag

```diff
@@ -71,30 +62,23 @@
                 })
             })
 
-            $('.queue-search')
-                .dropdown({
-                    apiSettings: {
-                        url: `${URLS.API}queues/?search={query}`,
-                    },
-                    clearable: true,
-                    preserveHTML: false,
-                    minCharacters: 2,
-                    fields: {
-                        remoteValues: 'results',
-                        name: 'name',
-                        value: 'id',
-                    },
-                    cache: false,
-                    //cache: false,
-                    maxResults: 5,
-                    onChange: (value, title) => {
-                        self.refs.queue.value = value
-                        self.form_updated()
-                    }
-                })
-            ;
-
-            self.get_available_queues()
+            $(self.refs.queue).dropdown({
+                // Note: Passing `public=true` so default behavior is users can search for public queues
+                apiSettings: {
+                    url: `${URLS.API}queues/?search={query}&public=true`,
+                },
+                clearable: true,
+                minCharacters: 2,
+                fields: {
+                    remoteValues: 'results',
+                    name: 'name',
```

Comment:
if `name` maps to `name`, I don't think we need this line?

Reply:
✔️